### PR TITLE
Added hunting zones based on polyzones

### PR DESCRIPTION
**Pull request by LeSiiN**
added Hunting Zones
Preview:  https://www.youtube.com/watch?v=1LMCgX7hHnk

### CODE REVIEW

File: config.lua [hunk 1]
```diff
@@ -35,6 +35,16 @@ Config.MinOffset = 1
 Config.MaxOffset = 120
 ---------------------------------------------------------------
 
+-- locations for the hunting zones ( Label: Name of Blip // Radius: Radius of the Alert and Blip)
+Config.Locations = {
+    ["hunting"] = {
+        [1] = {label = "Hunting Zone", radius = 250.0, coords = vector3(-1339.05, -3044.38, 13.94)},
+        --[2] = {label = "EXAMPLE 2", radius = 120.0, coords = vector3(-1481.19, 3517.79, 254.68)},
```
Comment (MonkeyWhisper) [inline config.lua]: Please remove 2nd line

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: client/cl_eventhandlers.lua [hunk 2]
```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
...
+        if Config.Debug then
+            local huntingzone = CircleZone:Create(vector3(hunting.coords.x, hunting.coords.y, hunting.coords.z), hunting.radius, {
+                name = Config.Locations["hunting"].label,
+                useZ = true,
+                debugPoly = true
+            })
+
+            huntingzone:onPlayerInOut(function (isPointInside)
+                if isPointInside then
+                    QBCore.Functions.Notify("DEBUG: INSIDE HUNTING AREA: "..hunting.label, "success")
```
Comment (MonkeyWhisper) [inline client/cl_eventhandlers.lua]: Please make this into a config for DebugHuntingZone true/false or just remove all together

File: client/cl_eventhandlers.lua [hunk 3]
```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
...
+                    print("inside")
+                else
+                    QBCore.Functions.Notify("DEBUG: OUTSIDE HUNTING AREA: "..hunting.label, "success")
```
Comment (MonkeyWhisper) [inline client/cl_eventhandlers.lua]: Please make this into a config for DebugHuntingZone true/false or just remove all together

File: client/cl_eventhandlers.lua [hunk 4]
```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
...
+                    print("outside")
```
Comment (MonkeyWhisper) [inline client/cl_eventhandlers.lua]: Remove prints please

File: client/cl_eventhandlers.lua [hunk 5]
```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
...
+            local huntingzone = CircleZone:Create(vector3(hunting.coords.x, hunting.coords.y, hunting.coords.z), hunting.radius, {
+                name = Config.Locations["hunting"].label,
+                useZ = true,
+                debugPoly = true
+            })
+
+            huntingzone:onPlayerInOut(function (isPointInside)
+                if isPointInside then
+                    QBCore.Functions.Notify("DEBUG: INSIDE HUNTING AREA: "..hunting.label, "success")
+                    print("inside")
```
Comment (MonkeyWhisper) [inline client/cl_eventhandlers.lua]: Remove prints please

File: client/cl_eventhandlers.lua [hunk 6]
```diff
@@ -9,6 +10,58 @@ local function isPlayerAWitness(witnesses)
...
+        SetBlipScale(blip, 0.8)
+        SetBlipColour(blip, 0)
+        SetBlipColour(huntingradius, 0)
+        SetBlipAlpha(huntingradius, 40)
+        BeginTextCommandSetBlipName("STRING")
+        AddTextComponentString(hunting.label)
+        EndTextCommandSetBlipName(blip)
+    end
+end)
+---------------------------------------------------------------------------------------
```
Comment (MonkeyWhisper) [inline client/cl_eventhandlers.lua:43]: Not needed, please remove

**Review by MonkeyWhisper — CHANGES_REQUESTED** [review]

File: client/cl_eventhandlers.lua (hunk 6 above)
Comment (LeSiiN) [reply client/cl_eventhandlers.lua:43]: What should i remove? The Blip stuff all together or the lines from 60 to 63 ?

File: config.lua (hunk 1 above)
Comment (LeSiiN) [reply config.lua]: Done

File: client/cl_eventhandlers.lua (hunk 2 above)
Comment (LeSiiN) [reply client/cl_eventhandlers.lua]: Removed all togehter

File: client/cl_eventhandlers.lua (hunk 3 above)
Comment (LeSiiN) [reply client/cl_eventhandlers.lua]: Removed all togehter

File: client/cl_eventhandlers.lua (hunk 4 above)
Comment (LeSiiN) [reply client/cl_eventhandlers.lua]: Removed all togehter

File: client/cl_eventhandlers.lua (hunk 5 above)
Comment (LeSiiN) [reply client/cl_eventhandlers.lua]: Removed all togehter

**Pull request merged**